### PR TITLE
chore: update RDS docs with newer PostgreSQL versions

### PR DESCRIPTION
**Pull request by Ephraim-G**
## Changes proposed in this pull request:
- updates the _docs/services/relational-database.md file to incorporate PostgreSQL versions 13,14 and 15 to be reflected on the relational-database page of cg-site.

## Security Considerations
N/A

### CODE REVIEW

**Comment by Ephraim-G** [comment]
in conjunction with https://github.com/cloud-gov/aws-broker/issues/281. Note that it doesn't specify anywhere on the RDS page the actual default version for PostgreSQL.

**Comment by markdboyd** [comment]
@Ephraim-G 

This isn't your fault, but I think this table is a bit misleading, because we aren't necessarily always offering the latest database version offered by AWS:

Screen Shot 2023-07-31 at 3 57 20 PM

For the `psql` plans in this plan, can we update the "Software Version" to say `15.x*` and for `mysql` plans to say `8.x.x*`?

Then below the table, can we add this code:

```
*Note: The actual database engine version used is the latest minor/patch version offered by AWS RDS for this major version*
```

And later on in the doc:

Screen Shot 2023-07-31 at 3 58 44 PM

Can we update the "Default value" to say: `PostgreSQL: 15.X, MySQL 8.x.x`?

File: _docs/services/relational-database.md [hunk 1]
```diff
@@ -15,30 +15,32 @@ If your application uses relational databases for storage, you can use the AWS R
 
 Plan Name                   | Description                                                                  | Software Version       |
```
Comment (markdboyd) [inline _docs/services/relational-database.md]:
```suggestion
Plan Name                   | Description                                                                  | Software Version (default)     |
```

File: _docs/services/relational-database.md [hunk 2]
```diff
@@ -15,30 +15,32 @@ If your application uses relational databases for storage, you can use the AWS R
...
+`medium-gp-mysql`           | Single-AZ RDS instance of MySQL, minimum 1 core, minimum 8 GiB memory                       | 8.x.x* |
+`medium-gp-mysql-redundant` | Multi-AZ RDS instance of MySQL, minimum 1 core, minimum 8 GiB memory             | 8.x.x* |
+`large-gp-mysql`            | Single-AZ RDS instance of MySQL, minimum 1 core, minimum 8 GiB memory                       | 8.x.x* |
+`large-gp-mysql-redundant`  | Multi-AZ RDS instance of MySQL, minimum 1 core, minimum 8 GiB memorye              | 8.x.x* |
+`xlarge-gp-mysql`           | Single-AZ RDS instance of MySQL, minimum 2 cores, minimum 16 GiB memory                      | 8.x.x* |
+`xlarge-gp-mysql-redundant` | Multi-AZ RDS instance of MySQL, minimum 2 cores, minimum 16 GiB memory            | 8.x.x* |
 `medium-oracle-se2`         | Single-AZ RDS instance of Oracle SE2, minimum 1 core, minimum 4 GiB memory                                           | AWS RDS Latest Default |
 
+*Note: The actual database engine version used is the latest minor/patch version offered by AWS RDS for this major version*
```
Comment (markdboyd) [inline _docs/services/relational-database.md]:
```suggestion
*\*Note: The actual database engine version used is the latest minor/patch version offered by AWS RDS for this major version*
```

File: _docs/services/relational-database.md [hunk 3]
```diff
@@ -106,7 +108,7 @@ Name               | Description
 ---                | ---                                                            | ---                    |
 `storage`          | Number of GB available to the database instance                | 10                     |
 `enable_functions` | Boolean to enable functions on supported databases             | false                  |
-`version`          | Specifies a supported major version in RDS (must be in quotes) | AWS RDS Latest Default |
+`version`          | Specifies a supported major version in RDS (must be in quotes) | PostgreSQL: 15.X, MySQL 8.x.x |
```
Comment (markdboyd) [inline _docs/services/relational-database.md]:
```suggestion
`version`          | Specifies a supported major version in RDS (must be in quotes) | PostgreSQL: `15`, MySQL `8.0` |
```

**Review by markdboyd — APPROVED** [review]

**Pull request merged**